mlrun/common/schemas/model_monitoring/model_endpoints.py

```diff
@@ -284,7 +284,14 @@ class ModelEndpointMonitoringMetric(BaseModel):
     app: str
     type: ModelEndpointMonitoringMetricType
     name: str
-    full_name: str
+    full_name: Optional[str] = None
+    kind: Optional[ResultKindApp] = None
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
Are you sure we need that *args? because the parent class `BaseModel` has no *args